vita3k/gui/src/settings_dialog.cpp

```diff
@@ -650,7 +650,7 @@ void draw_settings_dialog(GuiState &gui, EmuEnvState &emuenv) {
         ImGui::Spacing();
         ImGui::TextColored(GUI_COLOR_TEXT, "Enter button assignment \nSelect your 'Enter' button.");
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("This is the button that is used as 'Confirm' in applications dialogs. \nSome applications don't use this and get default confirmation button.");
+            ImGui::SetTooltip("This is the button that is used as 'Confirm' in applications' dialogs. \nSome applications don't use this and use the default confirmation button.");
```

Comment:
forgor to delete extra ' here

Reply:
it's a possessive apostrophe for multiple things so it needs a '
🤓

Reply:
fixed